src/lib.rs

```diff
@@ -887,6 +897,32 @@ mod tests {
         assert!(result.is_ok());
     }
 
+    #[test]
+    fn test_registration_packed_attestation() {
+        let wan_c = WebauthnEphemeralConfig::new(
+            "localhost:8443/auth",
+            "https://localhost:8443",
+            "localhost",
+        );
+        let mut wan = Webauthn::new(wan_c);
+
+        let chal = Challenge(
+            base64::decode_mode(
+                "lP6mWNAtG+/Vv15iM7lb/XRkdWMvVQ+lTyKwZuOg1Vo=",
+                base64::Base64Mode::Standard,
+            )
+            .unwrap(),
+        );
+
+        // Generated using Touch ID via Chrome
```

Comment:
Can you be more specific about the platform here? Touch ID on macos via chrome? Or Touch ID on a different version?

Reply:
I've used Touch ID on MacOS via Chrome, I'll add those details in the comment.

Reply:
I'm really curious about how you used touchid on chrome. Do you mind sending me some instructions? It sounds awesome.

Reply:
Yes this is very neat. I think it's working out of the box if you have a mac with TouchID hardware and an up to date browser. You can see a full demo here : https://drive.google.com/file/d/1IyPJ8X4MBOLLffut4yXXz1409s6ukNGC/view

Reply:
That's extremely cool, I do have a touchid mac, but I'm using FF not Chrome so I didn't know this was possible!